### PR TITLE
Update factory.go

### PR DESCRIPTION
**Pull request by YesterdayxD**
replace parameter type

### CODE REVIEW

File: pkg/controllers/job/plugins/factory.go [hunk 1]
```diff
@@ -40,7 +40,7 @@ var pluginBuilders = map[string]PluginBuilder{}
 type PluginBuilder func(pluginsinterface.PluginClientset, []string) pluginsinterface.PluginInterface
 
 // RegisterPluginBuilder register plugin builders
-func RegisterPluginBuilder(name string, pc func(pluginsinterface.PluginClientset, []string) pluginsinterface.PluginInterface) {
+func RegisterPluginBuilder(name string, PluginBuilder) {
```
Comment (hzxuzhonghu) [inline pkg/controllers/job/plugins/factory.go]: i think you need a param

Comment (YesterdayxD) [reply pkg/controllers/job/plugins/factory.go]: yes

**Pull request merged**